src/components/ui/sidebar/facility-nav.tsx

```diff
@@ -26,19 +27,50 @@ interface FacilityNavProps {
 
 function generateFacilityLinks(
   selectedFacility: UserFacilityModel | null,
-  t: TFunction,
-  permissions: {
-    canViewAppointments: boolean;
-    canListEncounters: boolean;
-    canCreateAppointment: boolean;
-    canCreateEncounter: boolean;
-    canViewEncounter: boolean;
-  },
+  links: NavigationLink[],
 ) {
   if (!selectedFacility) return [];
 
-  const baseUrl = `/facility/${selectedFacility.id}`;
```

Comment:
@shivankacker The idea was to preserve the baseUrl and not replace, since all URLs in the nav bar would have the baseUrl